pkgs/development/python-modules/xdot/default.nix

```diff
@@ -11,9 +11,17 @@ buildPythonPackage rec {
   };
 
   disabled = !isPy3k;
-
   nativeBuildInputs = [ wrapGAppsHook ];
-  propagatedBuildInputs = [ gobject-introspection pygobject3 graphviz gtk3 ];
+  propagatedBuildInputs = [ gobject-introspection pygobject3 graphviz gtk3 numpy ];
+  checkInputs = [ xvfb_run ];
+
+  checkPhase = ''
+    xvfb-run -s '-screen 0 800x600x24' ${python3.interpreter} nix_run_setup test
+  '';
+
+  # https://github.com/NixOS/nixpkgs/pull/107872#issuecomment-752175866
+  doCheck = false;
```

Comment:
Could this be changed to `doCheck = stdenv.isLinux;` instead, if the failure only happens on Darwin?

Reply:
Thanks lionello, I've [added that.](https://github.com/NixOS/nixpkgs/compare/8a3bf1a5e02250eca5e2b4329f50bdaa62af6cb7..2d091edabce03058078812a9cfb171b94614d122)